pystarport/cluster.py

```diff
@@ -33,7 +33,7 @@
     "autostart": "true",
     "autorestart": "true",
     "redirect_stderr": "true",
-    "startsecs": "3",
+    "startsecs": "1",
```

Comment:
what's this for?

Reply:
for fasten test, actually I'm not sure why need wait 3s

Reply:
Does it slowdown anything, I think the test logic just wait for port or block availability

Reply:
you is right